### DISCUSSION TITLE
How to identify what file is associated with a tab

### DISCUSSION THREAD

**Discussion by BobBagwill** [keepassxreboot/keepassxc#7541] [Q&A] [answered]
If I have a named database and I make a copy (which will end up with the same name) and open both, how do I tell which database file is associated with which tab?  Unnamed databases have the file name on the tab.  Thanks.

**Comment by droidmonkey** [comment, marked as answer]
Got to _Database_ -> _Database Settings_ and change the name of the database there. Leave it blank to have the file name show. You might as well go to the browser settings and reset the database uuid as well to prevent problems there.

**Reply by phoerious** [reply]
You can also simply hover over the tab. The file name will be in the tooltip popup.
